tdrs-backend/tdpservice/core/views.py

```diff
@@ -21,4 +26,15 @@ def write_logs(request):
                 f"{data['message']} "
                 f"for {request.user}")
 
+    if 'files' in data:
+        for file in data['files']:
+            LogEntry.objects.log_action(
+                user_id=User.objects.get(username=data['user']).pk,
+                content_type_id=ContentType.objects.get_for_model(ReportFile).pk,
+                object_id=file,
+                object_repr=str(data),
```

Comment:
It would be nice if we could format this data a little bit so it looks good in the logs. I am currently seeing this:
![Screen Shot 2021-05-03 at 1 15 44 PM](https://user-images.githubusercontent.com/1181427/116909095-b40b6800-ac11-11eb-8243-714002f42ab4.png)

Would like to see the file name and the report type added and have it formatted something like this:
```
type: alert
message: Submitted data file(s)
timestamp: 2021-05-03T17:08:34.639Z
user: csmith@goraft.tech
files: [1] requirements.txt
report type: Active Case Data
activity: upload
```

Reply:
Hmm it doesn't seem like Django Admin supports newlines in these table cells, but it does respect them on the Log Entry details. I'm thinking this might be acceptable though

![Screen Shot 2021-05-03 at 5 56 35 PM](https://user-images.githubusercontent.com/12901172/116938560-f09e8a00-ac38-11eb-8272-6b9f23e8ed73.png)

![Screen Shot 2021-05-03 at 5 57 10 PM](https://user-images.githubusercontent.com/12901172/116938593-fdbb7900-ac38-11eb-94b2-25cd9227ee08.png)

(note: I am aware the filename needs to be added to those^)

Reply:
Looks good. I think it's acceptable. We'll see what they think.